internal/conns/conns.go

```diff
@@ -1221,6 +1221,26 @@ func (c *Config) Client(ctx context.Context) (interface{}, diag.Diagnostics) {
 		}
 	})
 
+	client.ECSConn.Handlers.Retry.PushBack(func(r *request.Request) {
+		// By design the "WaitUntilServicesStable" method will poll every 15 seconds until a successful state
```

Comment:
This is not actually true, it polls until it's either MISSING, DRAINING, INACTIVE or successful, the docs aren't clear on this but the code is https://github.com/aws/aws-sdk-go/blob/main/service/ecs/waiters.go#L79-L105